ballerina/tests/asb_sender_receiver_negative_tests.bal

```diff
@@ -52,6 +55,46 @@ function testReceivePayloadWithIncorrectExpectedType() returns error? {
     groups: ["asb_sender_receiver_negative"],
     dependsOn: [testReceivePayloadWithIncorrectExpectedType]
 }
+function testInvalidComplete() returns error? {
```

Comment:
Shall we add another test case for invalid `abandon` invocation as well ?

Reply:
Sure. Addressed in [cae2e53](https://github.com/ballerina-platform/module-ballerinax-azure-service-bus/pull/196/commits/cae2e534ab7ec039301f791cdbade2f00a2f609d).